src/components/__tests__/LoginForm-test.js

```diff
@@ -12,125 +12,51 @@
 
 'use strict';
 
-jest.autoMockOff();
+jest.mock('tcomb-form-native', () => {
+  const React = require('React');
+  const t = require.requireActual('tcomb-form-native');
+  // Patch the base Component class to make rendering possible.
+  t.form.Component.prototype.render = function() {
```

Comment:
This method does some shady things and it doesn't work in the test renderer so I figured I'll patch it with a render method that exposes the props passed to it. That seems to work although there might be a better name that we can infer instead of `this.getTemplate().name`.